### DISCUSSION TITLE
[2021.08.24] 숫자 카드 2

### DISCUSSION THREAD

**Discussion by hyk339** [pyunji/Hyundai-ITnE-Team_QuadCore#33] [Algorithm]
[숫자 카드 2](https://www.acmicpc.net/problem/10816)

**Comment by pyunji** [comment]
[숫자 카드 2](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/BOJ/BOJ_10816.java)

**Reply by leechang4** [reply]
내일 설명 한번 부탁드립니다 ㅠㅠ

**Comment by hyk339** [comment]
[숫자 카드2 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/commit/d83f1686e994660cc74db9f0e8a0aa1debf5a5f7)

**Reply by leechang4** [reply]
내일 설명 한번 부탁드립니다 ㅠㅠ

**Comment by chaeyeon4u** [comment]
[숫자카드](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/Java/BOJ_10816.java)
